src/transformers/models/opt/modeling_tf_opt.py

```diff
@@ -84,7 +84,7 @@ def make_positions(mask, padding_idx: int):
 
     Position numbers begin at padding_idx+1. Padding symbols are ignored.
     """
-    positions = tf.cast(tf.math.cumsum(mask, axis=1), dtype=tf.int64) + padding_idx
+    positions = tf.math.cumsum(mask, axis=1) + padding_idx
```

Comment:
Could we align this with the new PT version. The padding idx is hardcoded so no need to have that be a parameter that can be passed.

Reply:
On it! 😊